contracts/contracts/lib/LookupKey.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+library LookupKey {
+    /**
+     * @notice Returns the key used to lookup authorizations
+     * @param ritualId The ID of the ritual
+     * @param encryptor The address of the encryptor
+     * @return The key used to lookup authorizations
+     */
+    function lookupKey(uint32 ritualId, address encryptor) internal pure returns (bytes32) {
```

Comment:
I don't think `encryptor` applies here anymore since this method is also used for administrator addresses, spender addresses etc. Should we use a more generic name eg. `anAddress` or `associatedAddress`  or ...

Reply:
It's true it's not strictly accurate here, but the hierarchy of roles/names is getting quite unwieldy. Maybe we can rename `encryptor` --> `encryptAdmin` and then rename this `admin` or `anAdmin`.

Reply:
Since this is a library code and is re-used in a couple of places, perhaps we should rename it to just `_address`. `encryptor` and `admin` are terms from specific contexts.